### PR TITLE
CRB-2374 & CB-20459

### PR DESCRIPTION
**Pull request by doktoric**
See detailed description in the commit message.

### CODE REVIEW

File: orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/knoxsso.xml.j2 [hunk 1]
```diff
@@ -133,9 +133,9 @@
         </param>
         {% endif %}
         <param>
-            <!-- 24 hours in milliseconds = 86400000 -->
+            <!-- 24 hours in milliseconds = 3600000 -->
```
Comment (lajosrodek) [inline orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/knoxsso.xml.j2:136]: `24 hours` -> `1 hour`

File: redbeams/src/main/resources/certs/aws/eu-south-2/0.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: rds-ca-2019-eu-south-2
```
Comment (lajosrodek) [inline redbeams/src/main/resources/certs/aws/eu-south-2/0.yml:1]: This should be `rds-ca-rsa2048-g1` as shown in the output of
```
aws --region eu-south-2 rds describe-certificates
```

File: redbeams/src/test/java/com/sequenceiq/redbeams/configuration/DatabaseServerSslCertificateConfigIntegrationTest.java [hunk 3]
```diff
@@ -185,6 +187,7 @@ void getSupportedPlatformsForCertsTest() {
                 Set.of(
                     "aws",
                     "aws." + REGION_EUS1,
```
Comment (lajosrodek) [inline redbeams/src/test/java/com/sequenceiq/redbeams/configuration/DatabaseServerSslCertificateConfigIntegrationTest.java:189]: Please also update all the other test cases in the class, adding verifications for the new region. You will have to add a constants `CERT_ISSUER_*` and `CLOUD_PROVIDER_IDENTIFIER_*` with the right expected values.

File: orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/knoxsso.xml.j2 (hunk 1 above)
Comment (lacikaaa) [inline orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/knoxsso.xml.j2:136]: I don't think so

**Review by lacikaaa — CHANGES_REQUESTED** [review]

**Review by lajosrodek — CHANGES_REQUESTED** [review]

**Comment by lacikaaa** [comment]
please in the future  follow the 1 commit/pr rule

**Comment by doktoric** [comment]
> DatabaseServerSslCertificateConfigIntegrationTest.java

**Comment by doktoric** [comment]
> please in the future follow the 1 commit/pr rule

I dont think we have such rule but let me do it

**Comment by lajosrodek** [comment]
Repost: #14121

**Pull request closed without merging**